network/src/counters.rs

```diff
@@ -92,7 +120,7 @@ pub static LIBRA_NETWORK_RPC_BYTES: Lazy<HistogramVec> = Lazy::new(|| {
 pub static LIBRA_NETWORK_RPC_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
     register_histogram_vec!(
         "libra_network_rpc_latency_seconds",
-        "Libra network rpc latency histogram",
+        "RPC request latency in seconds",
```

Comment:
what is a type? that seems redundant with protocol_id

Reply:
The dimensions are very confusing to me too.  That I think has to deal with refactoring the metrics in general (anything reading from the metric has to also get the name updated)